modules/tasks/taskSolutionFetchData.js

```diff
@@ -39,16 +35,9 @@ module.exports = Promise.method(async function fetchTaskSolutionData (solutionPa
       isPRMerged = true
     }
 
-    if (pullRequestData.title.includes('#')) {
-      const linkedPullRequestToIssueId = parseInt(pullRequestData.title.split('#')[1])
-      const githubIssueId = parseInt(task.url.split('/')[6])
-
-      // Verify if Issue is closed (first verify if the user accepted the assignment to the task)
-      if (task.dataValues.assigned && taskAssignment.dataValues.status === 'accepted') {
-        if (task.dataValues.provider === 'github' && task.dataValues.status === 'closed' && githubIssueId === linkedPullRequestToIssueId) {
-          isIssueClosed = true
-        }
-      }
```

Comment:
@alexanmtz 

I'm sorry but I don't get it. Could you please give me some help to understand?

Here's some description of the issue:

> When a user finished the PR and it is merged and the issue is closed, he can claim the bounty on Gitpay.

> We need to make a better check to validate that the issue is closed

> First, just by verifying that the issue on Gitpay is closed, it should be closed on Github. Add this verification to flag issue > closed on Github

> You should add the condition:

> ```
> if (task.dataValues.status === 'closed') {
>  isIssueClosed = true
> }
> ```

So, clarifying the description above, this conditional already exists, it's only together with other conditionals...

Questions:

1. If I maintain the existent conditionals as it is, the user must be assigned to the task **before** he tries to send the solution. If not, the flag related to the variable `isIssueClosed` will be false, preventing him to send the solution. What do you think?
2. If we gonna maintain the existent conditional as it is, do you still feel comfortable forcing the user who is the creator of the PR to follow our standard of PR titles? I'm asking because this is a standard for these existent conditionals, however, if we are planning to change it, it makes me wonder if this standard should still exist. Ex.: Every PR should have a hashtag following the issue number, like this -> <whenever_pr_name> #899